src/dashboard/public/styles/packs.css

```diff
@@ -79,11 +79,42 @@ main {
     }
 
     #comments {
-        word-wrap: break-word;
-        
+        div.commentInfo {
+            display: flex;
```

Comment:
text-align appears twice with different values, I assume it's not intentional

Reply:
Oops, sorry, I forgot to delete it when changing the text-align.